server/src/main/java/org/elasticsearch/common/util/BigArrays.java

```diff
@@ -451,7 +451,11 @@ public CircuitBreakerService breakerService() {   // TODO this feels like it is
             success = true;
         } finally {
             if (success == false) {
-                Releasables.closeWhileHandlingException(array);
+                try {
+                    array.close();
+                } catch (RuntimeException e) {
```

Comment:
Why not just `Exception` like we do everywhere else?

Reply:
This was a releasable already, but yea, made it it `Exception` as well :)